examples/intel_extension_for_pytorch/README.md

```diff
@@ -0,0 +1,179 @@
+# TorchServe with Intel® Extension for PyTorch*
+
+TorchServe can be used with Intel® Extension for PyTorch* (IPEX) to give performance boost on Intel hardware. 
```

Comment:
Can we specify which generation of hardware would most benefit from this

Reply:
SKX and later platforms (that have AVX512) benefit the most, but technically IPEX can help all platforms as there are some fusions and optimizations that are not yet available in upstream. IPEX can also be built for AVX2 and deployed as needed. Please see "Note" here https://github.com/otcshare/intel-extension-for-pytorch#intel-extension-for-pytorch